src/components/bid_card/BidCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { dateFormatter } from 'utilities/date';
+import { convertPenniesToDollars } from 'utilities/price';
+import './BidCard.scss';
+
+const BidCard = ({ timestamp, bid_price, userName, white }) => (
+  <section className={`bid--container ${white && 'bid--container-white'}`}>
```

Comment:
I think there's a better way to get a white background color than this - seems redundant to have a BidCard with class names `'bid--container bid--container--white'`. You should at least be able to do something like `bid--container white` where the white class just makes the background color white. When is a bid card background not white?

Reply:
the bid card in Item Page is white, but blue in the User Item Bids. I'll change the classname